src/arrayMethodFind.js

```diff
@@ -6,6 +6,12 @@
 function applyCustomFind() {
   [].__proto__.find2 = function(callback) {
     // write code here
+    for (let i = 0; i < this.length; i++) {
+      if (callback(this[i], i, this)) {
+
```

Comment:
if there is the first line of the new block, it's no need to set blank line before return

Reply:
Dima said tha we should always left a blank line before return